4.JQUERY/assets/js/sample.js

```diff
@@ -0,0 +1,89 @@
+// Declarations
+let First = document.querySelector(".Part1");
+let Second = document.querySelector(".Part2");
+let Third = document.querySelector(".Part3");
+let Fourth = document.querySelector(".Bonus");
+
+
+//Functions
+//Part 1
+const Part1=()=>{
+	let total=0;
+	for(let i=1; i <50 ; i+=2) {
+		total += i;
+	}
+	document.write(total);
+}
+
+// Part 2
+const Part2=()=>{
+	const Student ={
+		name:"Hillary",
+		age:14,
+		marks:[
+		100,
+		99,
+		98,
+		80,
+		70
+		]
+	}
+	let total = 0;
+	Student.marks.forEach((average)=> Average=total+=average/Student.marks.length);
+	document.write(`Average grade: ${Average} <br>`);
+	if(Average < 100){
+		document.write(`Grade : A`);
+	}
```

Comment:
The condition is lacking it will would always fall to the first condition if average is less than 100